src/block/notification/save.js

```diff
@@ -78,6 +78,7 @@ export const Save = props => {
 				{ attributes.isDismissible &&
 					<button
 						className="stk-block-notification__close-button"
+						{ ...applyFilters( 'stackable.notification.save.close-button', props.version ) }
```

Comment:
Check my suggestion for the progress circle, this prop would need to be added in by default, and then the deprecation filter would remove it. This is so that our save.js always contains our latest version, and the filters are there to modify it for older versions